purrr_basics/purrr_basics.Rmd

```diff
@@ -809,23 +1204,51 @@ class: middle, center
 
 ---
 
-## Your turn 9
+## `group_walk()`
+
+### Use `group_walk()` for side effects across groups 
+
+```{r group_walk}
+fs::dir_create(temp <- "temp")
+gapminder %>%
+  group_by(continent) %>%
+  group_walk(~ write_csv(x = .x, file = file.path(temp, 
```

Comment:
let's vary this a bit and do `group_walk(write_csv, [...the rest])`

Reply:
This is the only one I am fumbling with...I still can't convert it to the standard format. I am struggling on referencing `.data` in the `...` arguments. Would love to see how you got this to work.